### PR TITLE
Fetching episode specific information

### PR DESCRIPTION
**Pull request by kunalnpandey**
For Fetching Episode Specific Data

### CODE REVIEW

**Comment by kaustubhhiware** [comment]
Thanks @kunalnpandey  for sending this PR. I appreciate your contribution, but please get the issue assigned to yourself beforehand so that others know. Please keep that in mind for further contributions.

Could you give the sample usage and output for the code change you've made ?

**Comment by kunalnpandey** [comment]
episode status
Well I think I have handled the corner cases  which may generate errors and also have updated the README.

File: scrape_views.py [hunk 1]
```diff
@@ -302,3 +323,25 @@ def get_link(show, key, starturl):
...
             print( 'IMDB ratings average plot')
             average_plot(views, average)
+            
+    if not args.epi:
+        epi =""
+    else:
+        epi = args.epi
+        season_num=0
+        episode_num=0
+        if(len(epi)==6):
```
Comment (kaustubhhiware) [inline scrape_views.py]: This is an exceedingly complex method. We could go for a much simpler solution with this.

Given a string x.
Find the location of s/S => sindex
Find the location of e/E => eindex
```
season_num = int( x[sindex:eindex] )
episode_num = int( x[eindex+1:] ) # within try except
```

This way is much easier. How to find out the location of s/S is trivially searchable.

Comment (kunalnpandey) [reply scrape_views.py]: Well Sir now I have updated the code.

**Comment by kunalnpandey** [comment]
I have updated the code as you suggested.

**Comment by kunalnpandey** [comment]
Well now I think it will handle all the corner cases

**Comment by kaustubhhiware** [comment]
Great work !

I'll be squashing these commits while I merge it. Thank you for your contribution, I hope you keep contributing :smile:

**Comment by kunalnpandey** [comment]
😊😊
Sure Sir..!

**Comment by kunalnpandey** [comment]
I think now that issue that has been assigned to me has resolved.

**Pull request merged**